crates/criticalup-cli/src/cli/subcommand/init.rs

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: The Ferrocene Developers
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::cli::CommandExecute;
+use crate::errors::Error;
+use crate::Context;
+use clap::Parser;
+use criticalup_core::project_manifest::v1::sample_manifest;
+use tokio::fs::File;
+use tokio::io::{AsyncWriteExt, BufWriter};
+
+/// Create a manifest file (criticalup.toml) inside current directory
+#[derive(Debug, Parser)]
+pub(crate) struct Init {
+    /// Release version string of Ferrocene for this manifest
```

Comment:
As a user, how do I know what to put here? Is "25.02" sufficient? What about "stable"?

(Yes I know I'm being pedantic, perhaps point them to the docs)

Reply:
Just checked and we do need that index.html. https://releases.ferrocene.dev/ferrocene/ gives "access denied" even if I am logged in and can see https://releases.ferrocene.dev/ferrocene/index.html.